src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```
@@ -2,14 +2,18 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("login method was called");
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.warn("Username or password are incorrect");
         }
```

There's a critical security flaw here. If the password does not match, you should throw an `AuthenticationException` instead of just logging a warning. Currently, the method proceeds to return the user object, effectively granting access even with the wrong password.